lib/travis/build/script/csharp.rb

```diff
@@ -48,7 +61,7 @@ def export
         end
 
         def install
-          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution]
+          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution] && !is_mono_2?
         end
 
         def script
```

Comment:
If they override `script`, then this code won't get executed anyway, or am I missing something?